script/csharp/compile-command.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```
This should be bash if we're using `PIPESTATUS` (https://unix.stackexchange.com/questions/14270/get-exit-status-of-process-thats-piped-to-another).

Reply:
Okay, in that case both of them should be `bash`.